rosmon_core/src/main.cpp

```diff
@@ -121,7 +119,7 @@ void logToStdout(const rosmon::LogEvent& event)
 
 	clean.resize(len);
 
-	fmt::print("{:>20}: {}\n", event.source, clean);
+	fmtNoThrow::print("{:>20}: {}\n", event.source, clean);
```

Comment:
add the header, though its included transitively it makes sense to include it  as the function is explicitly called

Reply:
When I add the header there's a compile error that FMT_ENABLE_IF_T is defined more than once